.github/workflows/build.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         uses: arduino/setup-protoc@v1
         with:
           version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will hopefully prevent the rate limit build failures in GitHub actions.

Reply:
Fingers crossed!